O/OpenCL-Headers/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "OpenCL-Headers"
```

Comment:
The name must be a julia identifier, can't contain a dash

Reply:
Just noticed that as well.

https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=2513&view=logs&jobId=a5ce9d33-2463-5f1e-9da5-96efd60bf1fa&j=a5ce9d33-2463-5f1e-9da5-96efd60bf1fa&t=ec638f1c-e84c-549c-62fd-e8c8ba8e5f65